src/test/java/tests/SmokeTest2.java

```diff
@@ -0,0 +1,125 @@
+
+        HerokuauppMainPage contextMenu = new HerokuauppMainPage(browserService, true);
+        contextMenu.getContextMenuButton().click();
+
+        ContextMenuPage contextMenuPage = new ContextMenuPage(browserService, false);
+        WebElement element = waits.waitForVisibility(contextMenuPage.getContextMenuFrame());
+        Actions actions = new Actions(browserService.getDriver());
+        actions
+                .moveToElement(element)
+                .contextClick()
+                .build()
+                .perform();
+
+        Alert alert = browserService.getDriver().switchTo().alert();
+
+        Assert.assertEquals(alert.getText(), "You selected a context menu");
+
+    }
+
+    @Test
+    public void actionDynamicControlsTest() {
+        //Dynamic Controls
+
+        HerokuauppMainPage contextMenu = new HerokuauppMainPage(browserService, true);
+        contextMenu.getDynamicControlsButton().click();
+
+        // 1. Найти чекбокс
+        DynamicControlsPage dynamicControls = new DynamicControlsPage(browserService, false);
+        waits.waitForVisibility(dynamicControls.getCheckboxBox());
+        dynamicControls.getCheckboxBox().click();
+
+        // 2. Нажать на кнопку
+        dynamicControls.getRemoveButton().click();
+
+        // 3. Дождаться надписи “It’s gone”
+        waits.waitForVisibility(dynamicControls.getSuccessMessageText());
+        Assert.assertEquals(dynamicControls.getSuccessMessageCheckBox().getText(), "It's gone!");
+
+        // 4. Проверить, что чекбокса нет
+        Assert.assertFalse(dynamicControls.getAllCheckboxes().size() > 0);
+
+        // 5. Найти инпут
+        // 6. Проверить, что он disabled
+        Assert.assertFalse(dynamicControls.getInput().isEnabled());
+
+        // 7. Нажать на кнопку
+        dynamicControls.getEnableInputButton().click();
+
+        // 8. Дождаться надписи “It's enabled!”
+        waits.waitForVisibility(dynamicControls.getSuccessMessageInputText());
+        Assert.assertEquals(dynamicControls.getSuccessMessageInput().getText(), "It's enabled!");
+
+        // 9. Проверить, что инпут enabled
+        Assert.assertTrue(dynamicControls.getInput().isEnabled());
+    }
+
+    @Test
+    public void actionFileUploadTest() {
+//        Загрузить файл. Для относительного пути использовать
+//        https://stackoverflow.com/questions/45612089/how-to-upload-file-with-relative-path-in-selenium-webdriver
+//        Проверить, что имя файла на странице совпадает с именем загруженного файла
+
+        Actions actions = new Actions(browserService.getDriver());
+        HerokuauppMainPage contextMenu = new HerokuauppMainPage(browserService, true);
+        contextMenu.getFileUploadButton().click();
+
+        FileUploadPage fileUpload = new FileUploadPage(browserService, false);
+        String fileName = "vacation.png";
+        String filepath = System.getProperty("user.dir") + "\\" + fileName;
+
+        fileUpload.getUploadButton().sendKeys(filepath);
```

Comment:
org.openqa.selenium.InvalidArgumentException: invalid argument: File not found : /Users/aleksandr.trostyanko/Documents/Work/TechMeSkills/hw/rudak/AutomationTests\vacation.png